native/src/components/NavigationTile.tsx

```diff
@@ -50,7 +50,7 @@ type NavigationTileProps = {
 }
 
 const NavigationTile = ({ tile, width }: NavigationTileProps): ReactElement => (
-  <StyledPressable onPress={tile.onTilePress} width={width}>
+  <StyledPressable role='button' onPress={tile.onTilePress} width={width}>
```

Comment:
I think this is also a link, when tapping on a NavigationTile, you are sent to the route that you just tapped on.